### PR TITLE
PORTALS-2048

### PR DESCRIPTION
**Pull request by jay-hodgson**

### CODE REVIEW

File: src/lib/containers/table/StandaloneQueryWrapper.tsx [hunk 1]
```diff
@@ -0,0 +1,155 @@
+import * as React from 'react'
```
Comment (jay-hodgson) [inline src/lib/containers/table/StandaloneQueryWrapper.tsx:1]: Moved this heavily used component from portals to SRC.  See https://github.com/Sage-Bionetworks/portals/blob/master/src/portal-components/StandaloneQueryWrapper.tsx

File: src/lib/containers/table/StandaloneQueryWrapper.tsx [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+    link,
+    linkText,
+    title,
+    searchParams,
+    sqlOperator,
+    showAccessColumn,
+    sql,
+    hideDownload,
+    name,
+    showTopLevelControls = false,
```
Comment (jay-hodgson) [inline src/lib/containers/table/StandaloneQueryWrapper.tsx:74]: Added ability to show TopLevelControls for SynapseTable (PORTALS-2048)

File: src/lib/style/components/_query-wrapper-plot-nav.scss [hunk 3]
```diff
@@ -122,46 +122,6 @@
     margin-bottom: 10px;
   }
 
-  .TopLevelControls {
```
Comment (jay-hodgson) [inline src/lib/style/components/_query-wrapper-plot-nav.scss:125]: Used to be only under the Query Wrapper plot nav, but it now might be shown above a SynapseTable (in StandaloneQueryWrapper)

File: src/lib/containers/query_wrapper_plot_nav/TopLevelControls.tsx [hunk 4]
```diff
@@ -167,8 +174,9 @@ const TopLevelControls = (
           {controls.map(control => {
             const { key, icon, tooltipText } = control
             if (
-              key === 'showDownloadConfirmation' &&
-              (!isFileView || hideDownload)
```
Comment (jay-hodgson) [inline src/lib/containers/query_wrapper_plot_nav/TopLevelControls.tsx:171]: I think this was a bug introduced when implementing the Download Cart.  Download options are available for other Tables too (e.g. Export Table), you just can't add the query results to the Download Cart.  This case is already handled by the DownloadOptions component.

**Comment by jay-hodgson** [comment]
Screenshot of the StyleGuidist demo:
Screen Shot 2021-11-11 at 10 58 14 AM

File: src/lib/containers/table/StandaloneQueryWrapper.tsx [hunk 5]
```diff
@@ -0,0 +1,160 @@
...
+  const { data: entity } = useGetEntity(entityId)
+  return (
+    <QueryWrapper
+      {...rest}
+      initQueryRequest={derivedQueryRequestFromSearchParams}
+    >
+      {queryWrapperChildProps => {
+        return (
+          <>
+            {link && linkText ? (
```
Comment (jay-hodgson) [inline src/lib/containers/table/StandaloneQueryWrapper.tsx:102]: I don't understand why the output depends on link/linkText or title, but I didn't want to have to rework all of the portal-side configurations as part of this enhancement.

**Review by nickgros — APPROVED** [review]
Looks great!

**Pull request merged**